fedmsg/crypto/utils.py

```diff
@@ -98,7 +98,7 @@ def validate_policy(topic, signer, routing_policy, nitpicky=False):
             return True
 
 
-def _load_remote_cert(location, cache, cache_expiry, tries=0, **config):
+def _load_remote_cert(location, cache, cache_expiry, tries=1, **config):
```

Comment:
Why this change?

The docblock says

```
tries (int): The number of times to attempt downloading the certificate.
```

but it sounds like it's actually the number of *re*-tries upon failure, based on the name of the parameter being handed to requests. It would be good to clarify which it is in the docblock.